### ISSUE TITLE
"qrcode" dependency missing from requirements.txt

### ISSUE THREAD

**Issue by adpoliak** [drykath/mc-demo#1]
The mc-registration project requires the "qrcode" package be installed, but it does not get pulled in when pip retrieves the mc-registration project from Github

**Comment by drykath** [comment]
Yeah. :( Added it to this repo just to get it unblocked. Will continue to refine it, but at least it should be workable now!
